mne/viz/ica.py

```diff
@@ -1146,26 +1148,43 @@ def _plot_ica_overlay_raw(data, data_cln, times, title, ch_types_used, show):
     """
     import matplotlib.pyplot as plt
 
-    # Restore sensor space data and keep all PCA components
-    # let's now compare the date before and after cleaning.
-    # first the raw data
-    assert data.shape == data_cln.shape
-    fig, (ax1, ax2) = plt.subplots(2, 1, sharex=True)
-    plt.suptitle(title)
-    ax1.plot(times, data.T, color="r")
-    ax1.plot(times, data_cln.T, color="k")
-    ax1.set(xlabel="Time (s)", xlim=times[[0, -1]], title="Raw data")
-
-    _ch_types = {"mag": "Magnetometers", "grad": "Gradiometers", "eeg": "EEG"}
-    ch_types = ", ".join([_ch_types[k] for k in ch_types_used])
-    ax2.set_title("Average across channels ({})".format(ch_types))
-    ax2.plot(times, data.mean(0), color="r")
-    ax2.plot(times, data_cln.mean(0), color="k")
-    ax2.set(xlabel="Time (s)", xlim=times[[0, -1]])
-    tight_layout(fig=fig)
-
-    fig.subplots_adjust(top=0.90)
-    fig.canvas.draw()
+    _ch_type_names = {"mag": "Magnetometers", "grad": "Gradiometers", "eeg": "EEG"}
```

Comment:
Looks suspiciously like 

https://github.com/mne-tools/mne-python/blob/68e7272563fa88c2f0bd6d3e551eac9c51c2d07d/mne/defaults.py#L191-L194

:) So better to use `from ..defaults import DEFAULTS` then use `DEFAULTS['titles']`

Reply:
Definitely!